platforms/hanwha-wn/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+include(ExternalProject)
+
+set(BOOST_BUILD_PATH ${CMAKE_CURRENT_BINARY_DIR}/boost-prefix/src/boost)
+
+file(WRITE ${BOOST_BUILD_PATH}/user-config.jam "using gcc : arm : \"$ENV{WN}/arm-linux-gnueabi-g++\"")
+
+ExternalProject_Add(boost
+    URL https://sourceforge.net/projects/boost/files/boost/1.63.0/boost_1_63_0.tar.bz2
```

Comment:
For now we have minimum requirement for Boost in find_package(): it's 1.60.0
We may use whatever version suits us, I just used the latest one.